client/catapult/src/catapult/model/EntityRange.h

```diff
@@ -392,6 +392,9 @@ namespace catapult { namespace model {
 
 		/// Creates an entity range around \a numElements fixed size elements pointed to by \a pData.
 		static Range CopyFixed(const uint8_t* pData, size_t numElements) {
+			if (0 == numElements)
+				return Range();
```

Comment:
stringop-overread does not follow if in memcpy_cond so assumes copy is called when count is 0.